tests/qchem/of_tests/test_molecular_hamiltonian.py

```diff
@@ -1,14 +1,14 @@
-import os
 import sys
 
-import numpy as np
 import pytest
 
+from pennylane import Identity, PauliX, PauliY, PauliZ
+from pennylane import numpy as np
```

Comment:
why this change in this PR?

Reply:
We need pennylane numpy to make sure we can use `requires_grad` when building a Hamiltonian with `'dhf'` and the Pauli operators are needed to build the pennyane Hamiltonian. The original `test_molecular_hamiltonian` was only asserting: 
```
assert isinstance(built_hamiltonian, Hamiltonian)
assert qubits == 2 * nact_orbs
```